src/System.Private.ServiceModel/src/System/ServiceModel/UpnEndpointIdentity.cs

```diff
@@ -21,8 +22,23 @@ public UpnEndpointIdentity(string upnName)
         {
             if (upnName == null)
                 throw DiagnosticUtility.ExceptionUtility.ThrowHelperArgumentNull("upnName");
+#if SUPPORTS_WINDOWSIDENTITY
+            base.Initialize(Claim.CreateUpnClaim(upnName));
+#else 
+            throw ExceptionHelper.PlatformNotSupported("UpnEndpointIdentity is not supported on this platform");
+#endif // SUPPORTS_WINDOWSIDENTITY
```

Comment:
This was the line that caused the .NET Native build break. Claim.CreateUpnClaim was put under an #if SUPPORTS_WINDOWSIDENTITY.